package.json

```diff
@@ -48,7 +48,7 @@
     "test:ci": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
     "verify:docs": "npm run build:site -- --verify",
     "build:site": "./bin/build-docs-site.sh",
-    "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"examples/*/DIST/test/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
+    "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"examples/*/DIST/test/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\" --exclude packages/*/DIST/test/fixtures/**/*.js",
```

Comment:
The way the `mocha` script is run, it attempts run every JS files in the test folder. In this case, the exclude is necessary since reading the bad cyclic files (that are supposed to be read and caught in a test block) result in an error being thrown from the new code.

That being said, there might be a better approach to having mocha not read these files.

Reply:
I'm ok with the `exclude`, just asked cause I was curious why we needed it.